src/dvclive/report.py

```diff
@@ -55,17 +58,33 @@ def get_image_renderers(images_folder):
     return renderers
 
 
-def get_plot_renderers(plots_folder):
+def get_plot_renderers(plots_folder, live):
     renderers = []
     for suffix in SKLearnPlot.suffixes:
         for file in Path(plots_folder).rglob(f"*{suffix}"):
             name = file.stem
```

Comment:
For names like `train/roc`, I think it needs to be something like:

```suggestion
            name = str(file.relative_to(plots_folder).with_suffix(""))
```

This works, but maybe there's a cleaner solution.